File: Java/Vetores/Soma_vetores.java
package Java.Vetores;

import java.util.Locale;
import java.util.Scanner;


public class Soma_vetores {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);


        int N, i;
        
        
        System.out.print("Quantos valores vai ter cada vetor? ");
        N = sc.nextInt();

        int[] A = new int[N];
        int[] B = new int[N];
        int[] C = new int[N];
        
        System.out.println("Digite os valores do vetor A:");

        for (i = 0; i < N; i++) {
            System.out.print("Digite um numero: ");
            A[i] = sc.nextInt();
        }

        System.out.println("Digite os valores do vetor B:");
        for (i = 0; i < N; i++) {
            System.out.print("Digite um numero: ");
            B[i] = sc.nextInt();
        }
        
        for (i = 0; i < N; i++) {
        	C[i] = A[i] + B[i];
        }
        
        System.out.println("VETOR RESULTANTE:");
        
        for (i = 0; i < N; i++) {
            System.out.println(C[i]); 
        }
        
        

        sc.close();
 

    }
}

File: C++/Vetores/Soma_vetores.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int N, i, maiorPosicao;

    cout << "Quantos valores vai ter cada vetor ? ";
    cin >> N;

    int A[N], B[N], C[N];

    cout << "Digite os valores do vetor A: " << endl;

    for (i = 0; i < N; i++)
    {
        cin >> A[i];
    }

    cout << "Digite os valores do vetor B: " << endl;

    for (i = 0; i < N; i++)
    {
        cin >> B[i];
    }

    for (i = 0; i < N; i++)
    {
        C[i] = A[i] + B[i];
    }

    cout << "VETOR RESULTANTE: " << endl;

    for (i = 0; i < N; i++)
    {
        cout << C[i] << endl;
    }

    return 0;
}